_posts/2022-02-28-war-in-ukraine.md

```diff
@@ -13,14 +13,16 @@ humanitarian efforts.
 
 * If you know any Ukrainians, your coworkers in particular, talk to them and show your support. Even minor gestures are
   important. Ask directly what kind of help they might need rather than trying to guess.
-* Do not marginalize Russians and Belorussians. People are not the same as their governments, and often do not support
+* Do not marginalize Russians and Belarussians. People are not the same as their governments, and often do not support
   their states’ actions.
 * Work with your HR people and consider what kind of help you can offer as a company. Some examples would be extra days
   off, legal and financial support for bringing families to safety, or offering psychological support.
-* On a personal level, consider supporting one of the trustworthy humanitarian organizations. You can support them
-  with cash or by donating necessary items which are being collected in most cities.
 * Ask if the company can [donate directly, make donations by others easier](https://spolecznosc.allegro.pl/t5/blog/wsparcie-dla-ukrainy-co-mo%C5%BCesz-zrobi%C4%87-na-allegro-aby-pom%C3%B3c/ba-p/334998) [PL],
   provide means of communication, etc., depending on the industry.
+* On a personal level, consider supporting one of the trustworthy humanitarian organizations. You can support them
+  with cash, volunteer work, or donate necessary items which are being collected in most cities.
+  Be extremely cautious of fake fundraising campaigns that are popping up on every fundraising platform. Carefully
+  check the fundraiser.
 * If you work in tech, you are likely an authority on IT matters for your family and friends. Make them aware of
   the issue of disinformation, and encourage them to use only reliable information sources. Due to the threat of
   cyberattacks, encourage them to upgrade their systems and make backups.
```

Comment:
You can also warn them not to participate in DDoS attacks and not to install any software which claims to help in DDoS. It can be malware.